lib/ClientConnection.js

```diff
@@ -81,8 +81,10 @@ class ClientConnection {
             clearInterval(this.activityCheckInterval);
         }
 
-        if (error) {
-            this.log(this.server.LOGLEVEL.ERRORS, 'Closing connection with error: ', error);
+        if (code) {
+            const wasDisconnected = code === 1000 || code === 1001 || code === 1006;
+            const level = wasDisconnected ? this.server.LOGLEVEL.VERBOSE : this.server.LOGLEVEL.ERRRORS;
```

Comment:
@MtrXPL Can we log this at warning, I'm seeing that disconnects we ideally would want as errors (i.e. the target server hard powers off) are also logged at info. Striking the balance at warning may be best here.

Reply:
@rickkoliser Done